src/ir.rs

```diff
@@ -82,3 +82,17 @@ impl Aggregate {
         }
     }
 }
+
+#[derive(Clone, Debug, Serialize)]
+pub struct UdfCall {
+    pub id: String,
+    //TODO: Args may also be UDF calls
```

Comment:
So right now, args can only be envoy properties?

Reply:
Yes. Technically args are expressions, meaning they can involve arithmetic, method calls, integers, almost anything really. It depends on what we want to support.